mne/io/ctf/ctf.py

```diff
@@ -217,11 +217,11 @@ def _get_sample_info(fname, res4, system_clock):
     else:
         n_trial = n_samp // res4['nsamp']
         n_omit = n_samp_tot - n_samp
-        n_samp = n_trial * res4['nsamp']
```

Comment:
Ahh okay, makes sense. Then let's get rid of `n_trial` from the dict and we should be good.